legacy/src/main/java/org/opensearch/sql/legacy/plugin/RestSqlAction.java

```diff
@@ -66,228 +66,228 @@
 
 public class RestSqlAction extends BaseRestHandler {
 
-    private static final Logger LOG = LogManager.getLogger(RestSqlAction.class);
-
-    private final boolean allowExplicitIndex;
-
-    private static final Predicate<String> CONTAINS_SUBQUERY = Pattern.compile("\\(\\s*select ").asPredicate();
```

Comment:
could you elaborate more the change in RestSqlAction? I did not expect much change should in here.

Reply:
It's a white space change IntelliJ snuck in during when merging with 2.X. 

I'll redo the branch to discard those changes.

Reply:
Fixed in ea4588e3ca86157b709bd7184a27edc79253e8f5